Sources/TokamakDOM/App/App.swift

```diff
@@ -21,8 +21,15 @@ import TokamakCore
 import TokamakStaticHTML
 
 public extension App {
-  static func _launch(_ app: Self, _ rootEnvironment: EnvironmentValues) {
-    _launch(app, rootEnvironment, TokamakDOM.body)
+  static func _launch(_ app: Self, with configuration: _AppConfiguration) {
+    switch configuration.reconciler {
+    case .stack:
+      _launch(app, configuration.rootEnvironment, TokamakDOM.body)
+    case let .fiber(useDynamicLayout):
+//      _ = Unmanaged.passRetained(
```

Comment:
not sure if PR is ready for review, but if so, this commented code needs to be cleaned up